app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-core.js

```diff
@@ -52,6 +47,20 @@ window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {};
         }
         console.info(JSON.stringify(data, null, ' '))
       }
+
+      var ga4DisablingQueryStrings = ['smokey_cachebust', 'disable_ga4']
```

Comment:
I think you might be able to simplify this section with a regex, `[?&](disable_ga4|smokey_cachebust){1}` seems to work.

Reply:
@andysellick Great spot, I always forget about regex solutions. :+1: